### ISSUE TITLE
Output normalization/scale?

### ISSUE THREAD

**Issue by sipiseppala** [shicai/MobileNet-Caffe#37]
The pretrained model works fine but the output layer values (confidence levels, probabilities) seem to be between about **-10** and **+25**, with a typical top-1 result having value of 10...15. 

This is of course enough to determine the correct label from the highest value, but what is the absolute maximum range of the output layer's values?

E.g. if I want to output confidence level in percentage how do I scale the output?

**Comment by shicai** [comment]
add a softmax layer

**Comment by sipiseppala** [comment]
Ah, of course. Thanks!
